paperqa/core.py

```diff
@@ -71,14 +75,22 @@ async def map_fxn_summary(
     citation = text.name + ": " + text.doc.formatted_citation
     success = False
 
-    if prompt_runner:
-        llm_result = await prompt_runner(
-            {"question": question, "citation": citation, "text": text.text}
-            | (extra_prompt_data or {}),
-            callbacks,
-            "evidence:" + text.name,
+    if summary_llm_model and prompt_templates:
+        data = {"question": question, "citation": citation, "text": text.text} | (
+            extra_prompt_data or {}
+        )
+        message_prompt = prompt_templates[0]
+        system_prompt = prompt_templates[1]
```

Comment:
```suggestion
        message_prompt, system_prompt = prompt_templates
```

Two benefits here:
- We confirm the correct tuple length, otherwise it throws a `ValueError`
- Less lines

Reply:
Should we add any more information about the specific order in the doctrine? Currently it says
`prompt_templates: Tuple containing templates for the user and system prompts.`

Maybe edit it to:
```
prompt_templates: Tuple containing templates for the user and system prompts.
    prompt_templates = (user_prompt_templat, system_prompt_template)
```
?

Reply:
Yeah that sounds good. Also maybe start with "Optional two-tuple" to make it clear it's `| None` and the tuple has two entries